docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md

```diff
@@ -3,4 +3,67 @@ title = "Mixed Rules"
 weight = 6
 +++
 
-TODO
+* Configuration Item Explanation
+
```

Comment:
Hi, 

I suppose we need some introduction about this new section. At least we need to tell users, 
Hi, this is a mixed configuration example, and you can leverage all the features togather.
In order to xx, this page xxxx
Still, to scan the corresponding pages to learn more about the specific feature configuration.xxx